test/cpp/end2end/hybrid_end2end_test.cc

```diff
@@ -224,8 +224,9 @@ class HybridEnd2endTest : public ::testing::Test {
     bool ignored_ok;
     for (auto it = cqs_.begin(); it != cqs_.end(); ++it) {
       (*it)->Shutdown();
-      while ((*it)->Next(&ignored_tag, &ignored_ok))
+      while ((*it)->Next(&ignored_tag, &ignored_ok)) {
         ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done